electrum/lnpeer.py

```diff
@@ -2545,6 +2545,8 @@ def log_fail_reason(reason: str):
             return None, None
 
         # TODO check against actual min_final_cltv_expiry_delta from invoice (and give 2-3 blocks of leeway?)
+        # note: payment_bundles might get split here, e.g. one payment is "already forwarded" and the other is not.
+        #       In practice, for the swap prepayment use case, this does not matter.
```

Comment:
btw this made me realise that the description I wrote for bundled payments is incorrect:
https://github.com/spesmilo/electrum/blob/71255c1e735cdfcd881e045b9f2f6bc6b599f459/electrum/lnworker.py#L2312-L2316
It is hard to explain exactly what it means to bundle some RHASHes...